doc/man3/OCSP_resp_find_status.pod

```diff
@@ -105,10 +100,11 @@ signed the response are known via some out-of-band mechanism.
 OCSP_resp_get0_id() gets the responder id of I<bs>. If the responder ID is
 a name then <*pname> is set to the name and I<*pid> is set to NULL. If the
 responder ID is by key ID then I<*pid> is set to the key ID and I<*pname>
-is set to NULL. OCSP_resp_get1_id() leaves ownership of I<*pid> and I<*pname>
-with the caller, who is responsible for freeing them. Both functions return 1
-in case of success and 0 in case of failure. If OCSP_resp_get1_id() returns 0,
-no freeing of the results is necessary.
+is set to NULL.
+OCSP_resp_get1_id() leaves ownership of I<*pid> and I<*pname> with the caller,
```

Comment:
We usually treat separate paragraphs describing functions as stand alone.  What about:
> _OCSP_resp_get1_id() is the same as OCSP_resp_get0_id()
 but it leaves ownership of I<*pid> and I<*pname> with the caller, who is responsible for freeing them.  This function returns 1 on success and 0 on failure.  On failure, it is not necessary to freeing the result._

functions return 1 in case of success and 0 in case of failure.
 If OCSP_resp_get1_id() returns 0, no freeing of the results is necessary.

Reply:
Yeah, it's better to have a paragraph per function. Done.